openedx_events/enterprise/signals.py

```diff
@@ -24,11 +24,59 @@
 
 # .. event_type: org.openedx.enterprise.subsidy.redemption-reversed.v1
 # .. event_name: SUBSIDY_REDEMPTION_REVERSED
-# .. event_description: emitted when an enterprise subsidy is reversed.
+# .. event_description: (deprecated) emitted when an enterprise subsidy is reversed.
 # .. event_data: SubsidyRedemption
 SUBSIDY_REDEMPTION_REVERSED = OpenEdxPublicSignal(
     event_type="org.openedx.enterprise.subsidy.redemption-reversed.v1",
     data={
         "redemption": SubsidyRedemption,
     }
 )
+
+
+# .. event_type: org.openedx.enterprise_subsidies.ledger_transaction.created.v1
```

Comment:
According to the docs, the event_type should be:
`{Reverse DNS}.{Architecture Subdomain}.{Subject}.{Action}.{Major Version}`
Eg.:
- org.openedx.learning.course.enrollment.created.v1
- org.openedx.learning.student.registration.completed.v2
- org.openedx.learning.session.login.completed.v1

So this event type should be: org.openedx.enterprise.subsidy_ledger_transaction.created.v1 
Or a variation of it, but the domain should be enterprise.

Reply:
Sure, makes sense!

Reply:
Here's docs about it in case it helps: https://docs.openedx.org/projects/openedx-events/en/latest/decisions/0002-events-naming-and-versioning.html
More of each part of event types here: https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0041-arch-async-server-event-messaging.html#id5